index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/all.css">
+</head>
+<body>
+    <header>
+        <section class="toolbar-section">
+            <h2 class="logo">Akorede</h2>
+          
+            <nav class="header-nav">
+                <span class="toolbar-hamburger">
+                    <img src="images/hamburger.png" alt="">
+                </span>
+                <ul class="nav-link">
+                    <li><a href="">Portfolio</a></li>
+                    <li><a href="">About</a></li>
+                    <li><a href="">Contact</a></li>
+                </ul>
+            </nav>
+        </section>
```

Comment:
[OPTIONAL] [IMPORTANT] 
- [ ] Kindly make sure you don't have this `section` tag in your `header` since it's not semantic in this case. the `header` is supposed to be a stand-alone here.
- [ ] Also for semantic reasons, kindly make sure your `logo` is inside an anchor tag `<a href="#">Logo</a>` and not in a  heading `h2` for semantic reasons and as it is the standard for UX